src/library_webaudio.js

```diff
@@ -43,6 +43,7 @@ let LibraryWebAudio = {
   // dependency, because the user will not be able to utilize the node unless they call emscriptenGetAudioObject()
   // on it on JS side to connect it to the graph, so this avoids the user needing to manually do it on the command line.
   emscripten_create_audio_context__deps: ['$emscriptenRegisterAudioObject', '$emscriptenGetAudioObject'],
+  emscripten_create_audio_context__sig: 'ip',
```

Comment:
Looking at the code that is generated with this scheme, adding the `p` yields

```
options = Number(options);
...
```
although the ideal form for this would be that `options` would retain BigInt form, and instead the code line
```js
sampleRate: HEAP32[options+1] || void 0
```
would change to
```js
sampleRate: HEAP32[options+1n] || void 0
```
that would be better codegen for this instance.

For some instances, using an automatic `p` signature directive will yield the ideal code form, whereas for others, that is not the case. There is also the 4GB consideration, where we'd want to do
```js
options = options >>> 0;
```
here. Reading `convertPointerParams()`, it looks like it does not currently handle that scenario, although it could be expanded to do so.

LGTM, I'll need to try to find way to work on #15433.

Reply:
I agree there are probably things we can do better here for the wasm64 JS ABI, but hopefully they can be addressed centrally as part of the `p` signature handling and the `makeGetValue`/`makeSetValue` and other macros.    

For now, the ABI we are using is to cast all incoming bigint pointers to i53 numbers.. if there are more optimal forms hopefully we can convert to them without needing to change individual functions too much.